prov/efa/src/rdm/rxr_pkt_cmd.c

```diff
@@ -308,7 +308,7 @@ ssize_t rxr_pkt_post_one(struct efa_rdm_ep *efa_rdm_ep, struct efa_rdm_ope *ope,
 	/* If the send succeeded, the function rxr_pkt_entry_send will increase the
 	 * counter in efa_rdm_ep that tracks number of outstanding TX ops.
 	 */
-	err = rxr_pkt_entry_send(efa_rdm_ep, pkt_entry, flags);
+	err = rxr_pkt_entry_send(efa_rdm_ep, &pkt_entry, 1, flags);
```

Comment:
I feel like we should rename this function to rxr_pkt_entry_sendv, to match the libfabric and MPI naming convention

Reply:
sure. sounds good.

Reply:
addressed in new revision